server/android/src/android.rs

```diff
@@ -81,6 +83,7 @@ pub mod android {
     }
 
     #[no_mangle]
+    /// # Safety
```

Comment:
Is this just part of the TODO? or does this comment has a purpose?

Reply:
Part of above but removed unsafe and seems fine so deleting comments